src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -89,14 +109,116 @@ private static void openEditor(Player p, Block projector) {
+            BlockStorage.addBlockInfo(projector, Z_OFFSET_PARAMETER, String.valueOf(0.5));
+            openEditor(pl, projector);
+            return false;
+        });
+
+        // Y Axis
+        menu.addItem(2, new CustomItem(SkullItem.fromHash("b221da4418bd3bfb42eb64d2ab429c61decb8f4bf7d4cfb77a162be3dcb0b927"),
+            "&7Vertical Offset (+): &e" + DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER))),
+            "", "&eLeft Click: &7+0.1", "&eRight Click: &7+0.5"));
+        menu.addMenuClickHandler(2, (pl, slot, item, action) -> {
+            double yOffset = DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER)) + (action.isRightClicked() ? 0.5F : 0.1F));
+            double xOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), X_OFFSET_PARAMETER));
+            double zOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Z_OFFSET_PARAMETER));
+            ArmorStand hologram = getArmorStand(projector, true);
+            Location l = new Location(projector.getWorld(), projector.getX() + xOffset, projector.getY() + yOffset, projector.getZ() + zOffset);
+            hologram.teleport(l);
+
+            BlockStorage.addBlockInfo(projector, Y_OFFSET_PARAMETER, String.valueOf(yOffset));
+            openEditor(pl, projector);
+            return false;
+        });
+
+        menu.addItem(11, new CustomItem(SkullItem.fromHash("3b83bbccf4f0c86b12f6f79989d159454bf9281955d7e2411ce98c1b8aa38d8"),
+            "&7Vertical Offset (-): &e" + DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER))),
+            "", "&eLeft Click: &7+0.1", "&eRight Click: &7+0.5"));
+        menu.addMenuClickHandler(11, (pl, slot, item, action) -> {
+            double yOffset = DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER)) + (action.isRightClicked() ? -0.5F : -0.1F));
+            double xOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), X_OFFSET_PARAMETER));
+            double zOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Z_OFFSET_PARAMETER));
+            ArmorStand hologram = getArmorStand(projector, true);
+            Location l = new Location(projector.getWorld(), projector.getX() + xOffset, projector.getY() + yOffset, projector.getZ() + zOffset);
+            hologram.teleport(l);
+
+            BlockStorage.addBlockInfo(projector, Y_OFFSET_PARAMETER, String.valueOf(yOffset));
+            openEditor(pl, projector);
+            return false;
+        });
+
+
+        // X Axis
+        menu.addItem(4, new CustomItem(SkullItem.fromHash("f4628ace7c3afc61a476dc144893aaa642ba976d952b51ece26abafb896b8"),
```

Comment:
You should NOT put texture hashes in here, place them into the HeadTexture enum to make them reusable.